group_vars/all

```diff
@@ -28,7 +28,14 @@ galaxy_extras_apt_package_state: present
 tool_dependency_dir: /home/{{ galaxy_user_name }}/tool_dependencies
 shed_tools_dir: "{{ galaxy_server_dir }}/../shed_tools"
 tool_data_dir: "{{ galaxy_server_dir  }}/tool-data"
-additional_files_list: []
+miniconda_python: 2
+miniconda_version: "3.19.0"
+miniconda_installer_checksum: ""
+miniconda_prefix: "{{ tool_dependency_dir }}/_conda"
+
+additional_files_list:
```

Comment:
@mvdbeek I have a question about ansible:
here, if `additional_files_list` is (re-)attributed in a specific group_vars for instance in group_vars/kickstart with
```
additional_files_list:
  - { src: "extra-files/galaxy-kickstart/welcome.html", dest: "{{ galaxy_server_dir }}/static/" }
  - { src: "extra-files/galaxy-kickstart/galaxy-kickstart_logo.png", dest: "{{ galaxy_server_dir }}/static/images/" }
```
Then is appended or overwritten ?

If it is the second case, which I understand right now, then the reason why this PR is not working is there.
we should remove additional_files_list from this file and put back tool_sheds.conf.xml in every specific group_vars.